packages/core/src/unknown/UnknownEstimator.ts

```diff
@@ -16,7 +17,14 @@ export enum EstimateUnknownUsageBy {
   COST = 'cost',
   USAGE_AMOUNT = 'usageAmount',
 }
-
+const kwHdefault = {
+  service: {
+    usageUnit: {
+      cost: 50,
+      kilowattHours: 20,
+    },
+  },
```

Comment:
How did you land on these numbers as default values?

Reply:
@4upz I took them from the awsConstants object in the corresponding test file _packages/core/src/unknown/__tests__/UnknownEstimator.test.ts_ . Do you think this is an acceptable default?

Reply:
@abc516 Thanks for the clarification! Those may not be the best values to use as defaults. The values in the test file are usually mock data that is randomly chosen or selected fro ma specific sample group. We may want to rethink a more reasonable value that is retrieved elsewhere or exploring error handling if we can't derive a proper default to use